### ISSUE TITLE
Cannot build 2.0 release

### ISSUE THREAD

**Issue by eschnett** [agenium-scale/nsimd#83]
When I download the 2.0 release and follow the instructions, I receive this error:
```
$ bash scripts/build.sh for sse2 sse42 avx avx2 with gcc
+ set -e
+ SETUP_SH=/tmp/nsimd-2.0/scripts/setup.sh
+ NSCONFIG=/tmp/nsimd-2.0/scripts/../nstools/bin/nsconfig
+ HATCH_PY=/tmp/nsimd-2.0/scripts/../egg/hatch.py
+ BUILD_ROOT=/tmp/nsimd-2.0/scripts/..
+ sh /tmp/nsimd-2.0/scripts/setup.sh
+ set -e
+ NSTOOLS_DIR=/tmp/nsimd-2.0/scripts/../nstools
+ NSTOOLS_URL=git@github.com:agenium-scale/nstools.git
+ NSTOOLS_URL2=https://github.com/agenium-scale/nstools.git
+ '[' -e /tmp/nsimd-2.0/scripts/../nstools/README.md ']'
+ cd /tmp/nsimd-2.0/scripts/..
++ git remote get-url origin
++ sed s/nsimd/nstools/g
fatal: not a git repository (or any of the parent directories): .git
+ git clone
fatal: You must specify a repository to clone.
```

**Comment by gquintin** [comment]
What happens if you try this:

```sh
bash scripts/build.sh for sse2/sse42/avx/avx2 with gcc
```

?

**Comment by eschnett** [comment]
The same thing happens (but good catch, I forgot to add the slashes):
```
$ bash scripts/build.sh for sse2/sse42/avx/avx2 with gcc
+ set -e
+ SETUP_SH=/tmp/nsimd-2.0/scripts/setup.sh
+ NSCONFIG=/tmp/nsimd-2.0/scripts/../nstools/bin/nsconfig
+ HATCH_PY=/tmp/nsimd-2.0/scripts/../egg/hatch.py
+ BUILD_ROOT=/tmp/nsimd-2.0/scripts/..
+ sh /tmp/nsimd-2.0/scripts/setup.sh
+ set -e
+ NSTOOLS_DIR=/tmp/nsimd-2.0/scripts/../nstools
+ NSTOOLS_URL=git@github.com:agenium-scale/nstools.git
+ NSTOOLS_URL2=https://github.com/agenium-scale/nstools.git
+ '[' -e /tmp/nsimd-2.0/scripts/../nstools/README.md ']'
+ cd /tmp/nsimd-2.0/scripts/..
++ git remote get-url origin
++ sed s/nsimd/nstools/g
fatal: not a git repository (or any of the parent directories): .git
+ git clone
fatal: You must specify a repository to clone.

usage: git clone [<options>] [--] <repo> [<dir>]

    -v, --verbose         be more verbose
    -q, --quiet           be more quiet
    --progress            force progress reporting
    -n, --no-checkout     don't create a checkout
    --bare                create a bare repository
    --mirror              create a mirror repository (implies bare)
    -l, --local           to clone from a local repository
    --no-hardlinks        don't use local hardlinks, always copy
    -s, --shared          setup as shared repository
    --recurse-submodules[=<pathspec>]
                          initialize submodules in the clone
    --recursive ...       alias of --recurse-submodules
    -j, --jobs <n>        number of submodules cloned in parallel
    --template <template-directory>
                          directory from which templates will be used
    --reference <repo>    reference repository
    --reference-if-able <repo>
                          reference repository
    --dissociate          use --reference only while cloning
    -o, --origin <name>   use <name> instead of 'origin' to track upstream
    -b, --branch <branch>
                          checkout <branch> instead of the remote's HEAD
    -u, --upload-pack <path>
                          path to git-upload-pack on the remote
    --depth <depth>       create a shallow clone of that depth
    --shallow-since <time>
                          create a shallow clone since a specific time
    --shallow-exclude <revision>
                          deepen history of shallow clone, excluding rev
    --single-branch       clone only one branch, HEAD or --branch
    --no-tags             don't clone any tags, and make later fetches not to follow them
    --shallow-submodules  any cloned submodules will be shallow
    --separate-git-dir <gitdir>
                          separate git dir from working tree
    -c, --config <key=value>
                          set config inside the new repository
    --server-option <server-specific>
                          option to transmit
    -4, --ipv4            use IPv4 addresses only
    -6, --ipv6            use IPv6 addresses only
    --filter <args>       object filtering
    --remote-submodules   any cloned submodules will use their remote-tracking branch
    --sparse              initialize sparse-checkout file to include only files at root
```

**Comment by eschnett** [comment]
I'm building from the release tarball. There is no git repository. That's likely why the git command fails.

**Comment by gquintin** [comment]
Oh yes that's why. Good catch, I always do the build after a git-clone + checkout v2.0...

Will fix it right away !

**Comment by gquintin** [comment]
I fixed the bug, I will release a new version soon as I have fixed many bugs since 2.0.

**Comment by eschnett** [comment]
Yes please.

Another reason is that `cmake` as a build system is much more convenient than `nstools` when installing on various HPC systems, simply because `cmake` is a well known tool.
